libraries/matchers.rb

```diff
@@ -31,6 +31,7 @@
     oneview_fc_network:                 standard_actions + scope_actions + %i[reset_connection_template],
     oneview_fcoe_network:               standard_actions + scope_actions + %i[reset_connection_template],
     oneview_firmware:                   %i[add remove create_custom_spp],
+    oneview_hypervisor_cluster_profile: standard_actions + %i[create_or_update],
```

Comment:
i think it should be update here as per the actions defined

Reply:
@soodpr Corrected